sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/Microsoft.Azure.WebJobs.Extensions.Tables.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <NoWarn>$(NoWarn);SA1636;SA1649</NoWarn>
+		<WarningsNotAsErrors>$(WarningsNotAsErrors);NETSDK1206</WarningsNotAsErrors>
```

Comment:
The warning being produced is as follows:
> C:\Program Files\dotnet\sdk\8.0.111\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.targets(284,5): warning NETSDK1206: Found version-specific or di
stribution-specific runtime identifier(s): win7-x64. Affected libraries: Microsoft.Azure.DocumentDB.Core. In .NET 8.0 and higher, assets for version-
specific and distribution-specific runtime identifiers will not be found by default. See https://aka.ms/dotnet/rid-usage for details. [C:\Users\mredd
ing\source\repos\azure-sdk-for-net\sdk\tables\Microsoft.Azure.WebJobs.Extensions.Tables\tests\Microsoft.Azure.WebJobs.Extensions.Tables.Tests.csproj:
:TargetFramework=net8.0]

Microsoft.Azure.DocumentDB.Core is a transitive dependency brought in by Microsoft.Azure.WebJobs.Extensions.Storage 4.0.5. Newer versions of Microsoft.Azure.WebJobs.Extensions.Storage do not have a dependency on Microsoft.Azure.DocumentDB.Core, but it appears that this package is intentionally not using the latest in order to validate backwards compatibility.

Reply:
This is a test project & it looks like the live tests are passing so it seems to me like this warning is okay to be ignored.

Reply:
I'm seeing some test failures now but I think these are known failures